src/validation.cpp

```diff
@@ -107,7 +107,7 @@ CBlockIndex *pindexBestHeader = nullptr;
 Mutex g_best_block_mutex;
 std::condition_variable g_best_block_cv;
 uint256 g_best_block;
-int nScriptCheckThreads = 0;
+bool g_parallel_script_checks {false};
```

Comment:
1856073d6e8c774cc0a606275f264133884ef7bd

nit, is this a new format? The space makes sense when the initialization comes after an annotation but in this case we don't add the space.

Reply:
fixed